build/utils/dotnet/solution/solution_test.go

```diff
@@ -160,7 +160,7 @@ func TestLoad(t *testing.T) {
 	// 1. located in the project's root directory.
 	// 2. located in solutions directory.
 	solution := solution{path: filepath.Join(wd, "testdata", "nugetproj", "solutions"), slnFile: "nugetproj.sln"}
-	solutions, err := Load(solution.path, solution.slnFile, log)
+	solutions, err := Load(solution.path, solution.slnFile, "", log)
```

Comment:
in this test we can add a testcase for Load with an excluded pattern

Reply:
tests added